packages/radio-player/src/search-results-switcher.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  CSSResult,
+  TemplateResult,
+  PropertyValues,
+} from 'lit-element';
+
+@customElement('search-results-switcher')
+export default class SearchResultsSwitcher extends LitElement {
+  @property({ type: Number }) numberOfResults = 0;
+
+  @property({ type: Number }) currentResultIndex = 0;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <a @click=${this.goToPreviousResult}>&lt;</a>
+        ${this.currentResultIndex + 1} / ${this.numberOfResults}
+        <a @click=${this.goToNextResult}>&gt;</a>
+      </div>
+    `;
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      .container {
+        text-align: center;
+      }
+
+      a {
+        cursor: pointer;
+        -webkit-touch-callout: none;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+      }
+    `;
+  }
+
+  updated(changedProperties: PropertyValues): void {
+    if (changedProperties.has('numberOfResults')) {
+      this.currentResultIndex = 0;
+    }
+  }
+
+  goToPreviousResult(): void {
+    if (this.currentResultIndex < 1) {
```

Comment:
Might be more of a preference, but the way I've always achieved pagination looping is by incrementing/decrementing every call, then checking the value to see if it needs to wrap as one condition rather than two. Something like:
```javascript
this.currentResultIndex -= 1;
if (this.currentResultIndex < 0) {
  this.currentResultIndex = this.numberOfResults - 1;
}
```